linux/snowagent.config

```diff
@@ -27,72 +27,112 @@
     <!-- Configuration for File System scan-->
     <Software>
         <Include>
-            <Path recursive="true" unconditionally="false">/*</Path>
+			<!-- Only include necessary folders for the file scan-->
+            <Path recursive="true" unconditionally="false">/etc/*</Path>
+			<Path recursive="true" unconditionally="false">/var/*</Path>
+			<Path recursive="true" unconditionally="false">/bin/*</Path>
+			<Path recursive="true" unconditionally="false">/home/*</Path>
+			<Path recursive="true" unconditionally="false">/usr/*</Path>
+			<Path recursive="true" unconditionally="false">/opt/*</Path>
+			<Path recursive="true" unconditionally="false">/oracle/*</Path>
+			<Path recursive="true" unconditionally="false">/u01/*</Path>
         </Include>
         <IncludeCriteria enabled="true">
             <FileType>ELF*executable*</FileType>
         </IncludeCriteria>
         <Exclude>
-            <!-- Normal temporary directories on Unix systems. -->
-            <Path>/tmp*</Path>
-            <Path>/var/tmp*</Path>
-            <!-- Oracle audit files can be numerous. -->
-            <Path>*/oracle/*/audit*</Path>
-            <!-- Common non-relevant file extensions -->
-            <Path>*.au</Path>
-            <Path>*.bmp</Path>
-            <Path>*.bmu</Path>
-            <Path>*.cfg</Path>
-            <Path>*.class</Path>
-            <Path>*.conf</Path>
-            <Path>*.csm</Path>
-            <Path>*.css</Path>
-            <Path>*.dic</Path>
-            <Path>*.enc</Path>
-            <Path>*.gif</Path>
-            <Path>*.h</Path>
-            <Path>*.htm</Path>
-            <Path>*.html</Path>
-            <Path>*.jpg</Path>
-            <Path>*.js</Path>
-            <Path>*.log</Path>
-            <Path>*.mo</Path>
-            <Path>*.mof</Path>
-            <Path>*.packlist</Path>
-            <Path>*.pcf</Path>
-            <Path>*.pc</Path>
-            <Path>*.pf</Path>
-            <Path>*.pl</Path>
-            <Path>*.png</Path>
-            <Path>*.po</Path>
-            <Path>*.properties</Path>
-            <Path>*.rdf</Path>
-            <Path>*.sdl</Path>
-            <Path>*.so</Path>
-            <Path>*.sql</Path>
-            <Path>*.ttf</Path>
-            <Path>*.txe</Path>
-            <Path>*.txt</Path>
-            <Path>*.utf8</Path>
-            <Path>*.xml</Path>
-            <Path>*.xpt</Path>
-            <Path>*.zip</Path>
+          <!-- Exclude Filesystem scan -->
+		  <FileSystem>ext4</FileSystem>
```

Comment:
Why exclude ext4 filesystem, isn't that the default on most distros today?

Reply:
good catch

Reply:
If I remember correctly, by specifying these filesystems, they replace those disabled by default in the agent. Has this list been checked against that so that we don't enable a filesystem that is normally disabled?

Reply:
still don't understand, why to exclude ext4.

Reply:
Per is right. I have not checked that list of file systems. I will be removing this section entirely to avoid issues